### PR TITLE
ember data v1.0.0-beta.19 blogpost

### PR DESCRIPTION
**Pull request by fivetanley**

### CODE REVIEW

File: source/blog/2015-06-05-ember-data.1-0.beta.19-released.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+Ember Data 1.0.0-beta.19 contains a lot of bugfixes from the comunity!
+Thank you for helping us push toward a stable release of Ember Data!
+
+## Breaking Changes
+
+### Passing Classes to Store Methods
+
+Previously, you were allowed to pass a classes directly to store methods
```
Comment (igorT) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: This was deprecated though right?

File: source/blog/2015-06-05-ember-data.1-0.beta.19-released.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+var record = store.push('post', {id: 1});
+```
+
+To help you upgrade through this change easily, we've written an [Ember
+Watson command](https://github.com/abuiles/ember-watson#ember-watsonconvert-ember-data-model-lookups).
+You can use this command-line tool in both Ember-CLI projects and
+"globals mode" style projects.
+
+## Important Deprecations
+
```
Comment (igorT) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: Missing title

File: source/blog/2015-06-05-ember-data.1-0.beta.19-released.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Support for the Fixture Adapter has been removed. For a while, the
```
Comment (igorT) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: Deprecated

File: source/blog/2015-06-05-ember-data.1-0.beta.19-released.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+Fixture Adapter has been buggy and unmaintained. It has been deprecated
```
Comment (igorT) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: I would focus on it not being the right tool for the job even if it were not buggy

File: source/blog/2015-06-05-ember-data.1-0.beta.19-released.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+in Ember Data core, and will be removed in the 1.0 final release. The
+Fixture Adapter now lives as an
+[addon](https://github.com/emberjs/ember-data-fixture-adapter).
+
+## Internet Explorer 8
+
+This will be the last release to support Internet Explorer 8. Future
```
Comment (igorT) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: Probably will support IE8 until end of 1.0, we should say it is deprecated

File: source/blog/2015-06-05-ember-data.1-0.beta.19-released.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+versions of Ember Data will not support Internet Explorer 8.
+
+## Ember Data 1.0 and 2.0
+
+For the last year and a half, Ember Data has been in a "beta" state.
```
Comment (igorT) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: I would not mention this here, and have a more proper blog post announcing 2.0 and not hiding it this release

File: source/blog/2015-06-05-ember-data.1-0.beta.19-released.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
+---
+title: Ember Data v1.0.0-beta.19 Released
+author: Stanley Stuart
+tags: Recent Posts, Releases
+---
+
+Ember Data 1.0.0-beta.19 contains a lot of bugfixes from the comunity!
```
Comment (igorT) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: I thought we had some new features?

Comment (fivetanley) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: Nobody told me about new features ;)

File: source/blog/2015-06-05-ember-data.1-0.beta.19-released.md (hunk 1 above)
Comment (fivetanley) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: Nope, it uses an assertion.

File: source/blog/2015-06-05-ember-data.1-0.beta.19-released.md (hunk 2 above)
Comment (fivetanley) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: ? It has `## Important Deprecations`

Comment (igorT) [inline source/blog/2015-06-05-ember-data.1-0.beta.19-released.md]: it was missing the fixture adapter subtitle afai remember

**Pull request merged**